zerver/views/message_fetch.py

```diff
@@ -72,9 +71,11 @@ def get_search_fields(
     content_matches: Iterable[Tuple[int, int]],
     topic_matches: Iterable[Tuple[int, int]],
 ) -> Dict[str, str]:
+    with get_sqlalchemy_connection() as sa_conn:
+        escaped_topic_name = sa_conn.scalar(func.escape_html(topic_name))
```

Comment:
@timabbott @andersk let me how good is the idea to use the connection object each time we execute the `get_search_fields` function.

Reply:
No, don’t add new SQL queries. The *original* SQL queries in `zerver.lib.narrow` should be changed to add this additional column.

Reply:
done